helm/aws-load-balancer-controller/test.yaml

```diff
@@ -185,6 +185,9 @@ webhookNamespaceSelectors:
     values:
       - enabled
 
+# failurePolicy for webhooks. (default "Fail")
+webhookFailurePolicy:
```

Comment:
```suggestion
webhookFailurePolicy: "Fail"
```

Reply:
@wolvery @oliviassss sorry about the delay. Yes, adding the default is a great idea.